font/Cargo.toml

```diff
@@ -23,4 +23,5 @@ core-graphics = "0.17"
 core-foundation-sys = "0.6"
 
 [target.'cfg(windows)'.dependencies]
-dwrote = { version = "0.9.0" }
+dwrote = { git = "https://github.com/servo/dwrote-rs", rev = "c2486e8" }
```

Comment:
Just to note, I don't really mind in using git dependency on Windows for the time being.

Reply:
I do. We should usually stay away from git dependencies because we have no control over those retrospectively. With cargo we know immutability of versions is guaranteed.

It shouldn't be too hard to cooperate a release though.

Reply:
I know, I'm just saying that it's fine for me to see them on Windows tbh, since it's likely not that popular platform and support/cooperation could be hard. I don't want to see this mess like we had with winit/glutin on Linux tbh/macOS tbh.

Reply:
We've probably got over a thousand Windows users, so it shouldn't be taken lightly.